### PR TITLE
introduce ImageModifier to MySql source

### PR DESCRIPTION
**Pull request by stephane-airbyte**
introduce imageModifier to MySql so we don't use strings everywhere

### CODE REVIEW

**Comment by stephane-airbyte** [comment]
Current dependencies on/for this PR:
* master
  * **PR #32851** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/32851?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #32876** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/32876?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #32897** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/32897?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #32898** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/32898?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #32899** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/32899?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
            * **PR #32900** <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/32900?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/airbytehq/airbyte/32900?utm_source=stack-comment).

File: airbyte-integrations/connectors/source-mssql/src/test-integration/java/io/airbyte/integrations/source/mssql/AbstractSshMssqlSourceAcceptanceTest.java [hunk 1]
```diff
@@ -49,7 +51,7 @@ protected JsonNode getConfig() {
 
   @Override
   protected void setupEnvironment(final TestDestinationEnv environment) throws Exception {
-    testdb = MsSQLTestDatabase.in("mcr.microsoft.com/mssql/server:2017-latest", "withNetwork");
+    testdb = MsSQLTestDatabase.in(BaseImage.MSSQL_2017_LATEST, ImageModifier.NETWORK);
```
Comment (postamar) [inline airbyte-integrations/connectors/source-mssql/src/test-integration/java/io/airbyte/integrations/source/mssql/AbstractSshMssqlSourceAcceptanceTest.java:54]: These MS SQL Server changes seem to have made their way into this PR by mistake.

**Review by postamar — COMMENTED** [review]
This PR seems to have changes for all 3 connectors, which is I'm guessing not intentional.

File: airbyte-integrations/connectors/source-mysql/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/CdcMySqlSourceAcceptanceTest.java [hunk 2]
```diff
@@ -102,16 +103,16 @@ protected JsonNode getState() {
...
         .withCdcPermissions()
         .with("CREATE TABLE id_and_name(id INTEGER, name VARCHAR(200));")
         .with("INSERT INTO id_and_name (id, name) VALUES (1,'picard'),  (2, 'crusher'), (3, 'vash');")
         .with("CREATE TABLE starships(id INTEGER, name VARCHAR(200));")
         .with("INSERT INTO starships (id, name) VALUES (1,'enterprise-d'),  (2, 'defiant'), (3, 'yamato');");
   }
 
-  protected Stream<String> extraContainerFactoryMethods() {
-    return Stream.empty();
+  protected ContainerModifier[] getImageModifiers() {
```
Comment (postamar) [inline airbyte-integrations/connectors/source-mysql/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/CdcMySqlSourceAcceptanceTest.java:114]: s/getImageModifiers/getContainerModifiers/

**Review by postamar — APPROVED** [review]

**Comment by stephane-airbyte** [comment]
### Merge activity

* **Dec 7, 2:39 PM**: @@stephane-airbyte started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/airbytehq/airbyte/32900).

**Pull request merged**